build/scraper.mjs

```diff
@@ -133,10 +134,6 @@ class Scraper {
    *    identifier corresponding to a {@link module:warframe-items.Item.uniqueName|uniqueName}
    * @property {string} textureLocation location of the image for the corresponding item
    */
-  /**
-   * @typedef {Object} ImageManifest
-   * @property {ImageManifestItem[]} Manifest
-   */
```

Comment:
this is an internal type def and should not be removed

Reply:
still waiting on this to come back

Reply:
gimme a few lost my gpg keys

Reply:
:fearful:

Reply:
Line should be back now? Also I thought I did, forgot I put the private and public key in the yubi